lib/ui/screens/pin/pin_screen.dart

```diff
@@ -70,7 +69,7 @@ class _PinScreenContentState extends State<PinScreenContent> {
           _errorMessage = null;
         }
 
-        _errorMessage = _errorMessage ?? _extractErrorMessage(state);
+        _errorMessage = _extractErrorMessage(state);
```

Comment:
❓ I'm not sure why the error message was only updated if it was not yet set, but it caused the first error message to stay displayed even if the original error was fixed and a new one was triggered. But I don't want to just merge this change without asking if there were a reason for this logic here @tamas-toth @laszlo-domonkos ?

Reply:
Okay, I see now, the missing piece was not that it was set to a value in case of local validation failed, but that when it succeeded, it was reset to being null, and then the "extraction" from state happened here. This finally confirms to me though that it's okay that I removed this logic, however I think I can also than remove the _errorMessage property as well and only have a local variable for it.

I'll also remove the translation than, thanks for pointing it out!